src/handler/make_handler.jl

```diff
@@ -40,14 +40,83 @@ validate_layout(layout::Component) = validate(layout)
 validate_layout(layout::Function) = validate_layout(layout())
 
 validate_layout(layout) = error("The layout must be a component, tree of components, or a function which returns a component.")
+
+function stack_from_callback(full_stack)
+    result = Base.StackTraces.StackFrame[]
+    it = iterate(full_stack)
+    while !isnothing(it) && it[1].func != :process_callback_call
+        push!(result, it[1])
+        it = iterate(full_stack, it[2])
+    end
+    return result
+end
+
+
+function exception_handling(ex)
+    st = stack_from_callback(stacktrace(catch_backtrace()))
+    @error exception = (ex, st)
+    return HTTP.Response(500)
+end
+
+#<!DOCTYPE HTML><html><body><pre><h3>%s</h3><br>Error: %s: %s<br>%s</pre></body></html>
+# ### DashR Traceback (most recent/innermost call last) ###
+function debug_exception_handling(ex)
+    response = HTTP.Response(500, ["Content-Type" => "text/html"])
+    io = IOBuffer()
+    write(io,
+         "<!DOCTYPE HTML><html>",
+         "<head>",
+         "<style type=\"text/css\">",
+         """
+            div.debugger { text-align: left; padding: 12px; margin: auto;
+            background-color: white; }
+            h1           { font-size: 36px; margin: 0 0 0.3em 0; }
+            div.detail { cursor: pointer; }
+            div.detail p { margin: 0 0 8px 13px; font-size: 14px; white-space: pre-wrap;
+            font-family: monospace; }
+            div.plain { border: 1px solid #ddd; margin: 0 0 1em 0; padding: 10px; }
+            div.plain p      { margin: 0; }
+            div.plain textarea,
+            div.plain pre { margin: 10px 0 0 0; padding: 4px;
+                background-color: #E8EFF0; border: 1px solid #D3E7E9; }
+            div.plain textarea { width: 99%; height: 300px; }
+            """,
+         "</style>",
+         "</head>",
+         "<body style=\"background-color: #fff\">",
+         "<div class=\"debugger\">",
+         "<div class=\"detail\">",
+         "<p class=\"errormsg\">")
+    showerror(io, ex)
+    write(io,
+        "</p>",
+        "</div>",
+        "</div>",
+        "<div class=\"plain\">",
+        "<textarea cols=\"50\" rows=\"10\" name=\"code\" readonly>"
+    )
+    st = stack_from_callback(stacktrace(catch_backtrace()))
+    Base.show_backtrace(io, st)
+
+    write(io, "</textarea>")
```

Comment:
Looks really good! Two thoughts:
- Is it important to use a `<textarea>` rather than `<pre>`? I see you have it tagged as `readonly`, but it still kind of has the look of an input.
- Can we repeat the error message either above or below the stack trace (next to the line that actually failed)? Maybe people working consistently in a single language don't have this issue, but different languages don't agree on whether you're stacking up or down, so I find it helpful to have the error message as a cue for which way to read the stack.

Reply:
@alexcjohnson 
* As for textarea, there is something similar in the Python version. textarea is very copy-friendly - all variations of "select-all" (double-click, cmd-a and etc) selects entire trace witout any surrounding tags

* Do you mean to add  call of `process_callback_call` function from which a specific handler is called  to the trace?

Reply:
- textarea - ok, that's fine. I see `<pre>` in the standard Python version but if you find `<textarea>` more useful we can go with it.
- repeated message: I'm imagining something like:
```
BoundsError: attempt to access 0-element Array{Any,1} at index [10]
```

```
BoundsError: attempt to access 0-element Array{Any,1} at index [10]
Stacktrace:
 [1] getindex at ./array.jl:744 [inlined]
 [2] (::Main.##422.var"#14#29")(::Array{Any,1}) at /Users/war/julia_projects/Dash/tmp/debug.jl:101
```

As a way to make it more obvious that the error came from line [1] rather than line [2] (?). For comparison here's what an error looks like in Python, and you can see the error message itself is repeated *after* the stack trace because it's actually the last item in the stack trace that failed (because the innermost function call is at the bottom):
<img width="605" alt="Screen Shot 2020-08-07 at 1 59 47 PM" src="https://user-images.githubusercontent.com/2678795/89674635-795ce500-d8b6-11ea-8656-d8a2c392481c.png">
As opposed to JS where the error message is shown at the top, and the deepest function call is at the top:
<img width="223" alt="Screen Shot 2020-08-07 at 2 04 09 PM" src="https://user-images.githubusercontent.com/2678795/89674954-10c23800-d8b7-11ea-9bc1-4f6a0c2b6980.png">

Reply:
@alexcjohnson done at 76af75b